src/relay/pass/enable_tensorrt.cc

```diff
@@ -0,0 +1,585 @@
+          attrs->axes[1].as<IntImm>()->value == 0) {
+        auto* multiply = n->args[0].as<CallNode>();
+        if (multiply && multiply->op.same_as(multiply_op)) {
+          const ConstantNode* const_1 = multiply->args[1].as<ConstantNode>();
+          if (const_1 && const_1->is_scalar() &&
+              *static_cast<float*>(const_1->data->data) == 1.0f) {
+            auto* transpose = multiply->args[0].as<CallNode>();
+            if (transpose && transpose->op.same_as(transpose_op)) {
+              const auto* attrs = n->attrs.as<TransposeAttrs>();
+              if (attrs->axes.defined() && attrs->axes.size() == 2 &&
+                  attrs->axes[0].as<IntImm>()->value == 1 &&
+                  attrs->axes[1].as<IntImm>()->value == 0) {
+                return transpose->args[0];
+              }
+            }
+          }
+        }
+      }
+    }
+    return ExprMutator::VisitExpr_(n);
+  }
+};
+
+// Base TRT ops.
+static const std::unordered_set<std::string> trt_base_compatible_ops = {
+    {"nn.dropout"},  // Will be removed by EnableTrt pass.
+    {"nn.relu"},
+    {"sigmoid"},
+    {"tanh"},
+    {"nn.batch_norm"},
+    {"nn.softmax"},
+    {"nn.conv2d"},
+    {"nn.dense"},
+    {"nn.bias_add"},
+    {"add"},
+    {"subtract"},
+    {"multiply"},
+    {"divide"},
+    {"power"},
+    {"nn.max_pool2d"},
+    {"nn.avg_pool2d"},
+    {"nn.global_max_pool2d"},
+    {"nn.global_avg_pool2d"},
+    {"exp"},
+    {"log"},
+    {"sqrt"},
+    {"abs"},
+    {"negative"},
+    {"nn.batch_flatten"},
+    {"expand_dims"},
+    {"squeeze"},
+    {"concatenate"},
+    {"nn.conv2d_transpose"},
+    {"transpose"},
+    {"reshape"},
+    {"nn.pad"},
+    {"sum"},
+    {"prod"},
+    {"max"},
+    {"min"},
+    {"mean"},
+    {"contrib.adaptive_max_pool2d"},
+    {"contrib.adaptive_avg_pool2d"}};
+
+// Ops which require TRT 5.1.5+
+static const std::unordered_set<std::string> trt_5_1_5_compatible_ops = {
+    {"clip"}, {"nn.leaky_relu"}, {"sin"},   {"cos"},
+    {"atan"}, {"ceil"},          {"floor"}, {"strided_slice"}};
+
+// Ops which require TRT 6.0.1+
+static const std::unordered_set<std::string> trt_6_0_1_compatible_ops = {
+    {"image.resize"}};
+
+bool TrtVersionGe(const std::tuple<int, int, int>& curr_version, int major,
+                  int minor, int patch) {
+  if (std::get<0>(curr_version) > major) return true;
+  if (std::get<0>(curr_version) == major && std::get<1>(curr_version) > minor)
+    return true;
+  if (std::get<0>(curr_version) == major &&
+      std::get<1>(curr_version) == minor && std::get<2>(curr_version) >= patch)
+    return true;
+  return false;
+}
+
+std::vector<int> GetShape(const Type& type) {
+  const auto* ttype = type.as<TensorTypeNode>();
+  CHECK(ttype);
+  std::vector<int> _shape;
+  for (size_t i = 0; i < ttype->shape.size(); ++i) {
+    auto* val = ttype->shape[i].as<IntImm>();
+    CHECK(val);
+    _shape.push_back(val->value);
+  }
+  return _shape;
+}
+
+class TrtChecker : public ExprVisitor {
```

Comment:
This function can be remove once https://github.com/apache/incubator-tvm/pull/4570 is merged. Maybe add a todo for now. This could be more to the python template.